vars.json

```diff
@@ -15,6 +15,13 @@
   },
   "versions": {
     "current": [
```

Comment:
Why does this list say `current`, if it's in reality contains `old`/`previous` versions?